task/gradle/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- crshnburn
+reviewers:
+- crshnburn
```

Comment:
to add yourself as the owner of this task, you need to sign the CLA first and then add yourself in the Tekton Org. Can you please do the needful?

Reply:
https://github.com/tektoncd/community/pull/621 raised to add myself to the org.